src/utils/file.js

```diff
@@ -64,3 +72,34 @@ export default async (src: string, auth: Auth): Promise<mixed> => {
     }),
   );
 };
+
+/**
+ * Downloads any generic resource on the internet to the user's device. Fails
+ * if we do not have permission.
+ *
+ * @param url Complete URL to the file. Can be any location on the internet.
+ */
+export const downloadFile = async (url: string): Promise<mixed> => {
+  if (Platform.OS === 'ios') {
+    return CameraRoll.saveToCameraRoll(url);
```

Comment:
On iOS, I think the camera roll is meant for images and videos only? Dowloading the `.docx` file from your [message in `#test here`](https://chat.zulip.org/#narrow/stream/7-test-here/topic/testtest/near/869537) didn't work when I tested it, at least on the iOS simulator. There was no visible change to the screen when I pressed the link.